pom.xml

```diff
@@ -142,28 +142,6 @@
         <project.previous.version>9.2.0</project.previous.version>
     </properties>
 
-    <pluginRepositories>
```

Comment:
have you tested this with eclipse? That the project clean installs.
I think this was put in specifically for eclipse users.

Reply:
I did not test in eclipse. I'd like to understand why it would affect eclipse though, since the pluginRepositories section was edited relatively recently and we've supported eclipse for years.

Reply:
I can try this out in Eclipse shortly.